doc/api/deprecations.md

```diff
@@ -2336,6 +2335,19 @@ Type: Runtime
 Setting the TLS ServerName to an IP address is not permitted by
 [RFC 6066][]. This will be ignored in a future version.
 
+<a id="DEP0XXX"></a>
+### DEP0XXX: using REPLServer#rli
```

Comment:
I am not sure if `#` is common here. This doc usually uses `.prototype.` in such cases, while `#` is used only once (`Buffer#(read|write) operations`). Feel free to ignore if we can treat it as a common sign in a broader context.